src/Main.scss

```diff
@@ -52,4 +53,8 @@ body,
     width: 100%;
     height: 100%;
     align-items: flex-start;
+}
+
+.flex-layout {
```

Comment:
Zdaj da se boš naučil tudi imena pravilno dajati, imitiraj tailwind css class-e. Poglej kako ime oni uporabijo za display: flex pa njihovo uporabi. Pa to naj velja za vse te one linerje ki jih boš delal.